manifest.json

```diff
@@ -3,23 +3,29 @@
   "name": "No LinkedIn News",
   "version": "1.0",
   "description": "Disable LinkedIn News on home page and in news section",
-   "permissions": ["activeTab", "tabs"],
+   "permissions": ["activeTab", "tabs", "scripting"],
   "icons": {
     "16": "images/newspaper-emoji.png",
     "48": "images/newspaper-emoji.png"
 },
 "background": {
     "service_worker": "scripts/background.js"
 },
+"web_accessible_resources": [
```

Comment:
Added this so that these files would be exposed in order to be loaded into the web page.  See [this article](https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/) for more information.